docs/pages/operations-guide.md

```diff
@@ -215,11 +247,11 @@ Agent 2: Y Y
 Agent 3: X B
 ```
 
-Configuring `ROOT` vs `MOUNT` volumes may depend on the service. Some services will support customizing this setting when it's relevant while others may assume one or the other.
+Configuring `ROOT` vs `MOUNT` volumes may depend on the service. Some services will support customizing this setting when it is relevant, while others may assume one or the other.
 
 ## Pods vs Tasks
 
-A Task generally maps to a process. A Pod is a collection of Tasks which share an environment with each other. In practice, all Tasks in a Pod will come up and go down together, hence why [restart](#restart-a-pod)/[replace](#replace-a-pod) operations are at Pod granularity rather than Task granularity.
+A Task generally maps to a process. A Pod is a collection of Tasks that share an environment. All Tasks in a Pod will come up and go down together. Therefore, [restart](#restart-a-pod)/[replace](#replace-a-pod) operations are at Pod granularity rather than Task granularity.
```

Comment:
"restart and replace" instead of "restart/replace" would probably read better here